scipy/optimize/tests/test_minpack.py

```diff
@@ -517,6 +517,26 @@ def test_maxfev_and_bounds(self):
         assert_allclose(popt1, 2, atol=1e-14)
         assert_allclose(popt2, 2, atol=1e-14)
 
+    def test_curvefit_covariance(self):
```

Comment:
I noticed that scipy.linalg and numpy.linalg behave differently. 

Contrast:
http://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.cholesky.html

with 

http://scipy.github.io/devdocs/generated/scipy.linalg.cholesky.html#scipy.linalg.cholesky

That is really weird. I will fix this problem in a couple of hours and write a proper unit test for checking this.